tests/test_make.py

```diff
@@ -28,3 +34,32 @@ def test_nek5000_make(pytestconfig):
         ]
         for future in as_completed(simultaneous_builds):
             assert future.result()
+
+
+def test_snek_make_help():
+    with patch.object(sys, "argv", ["snek-make", "-h"]):
+        with pytest.raises(SystemExit):
+            snek_make()
+
+
+@contextmanager
+def current_dir_changed_to(path):
+    cwd = Path.cwd()
+    os.chdir(path)
+    try:
+        yield
+    finally:
+        os.chdir(cwd)
```

Comment:
This is fine, but we could have used `monkeypatch.chdir` from pytest, instead of this.

https://docs.pytest.org/en/stable/how-to/monkeypatch.html#chdir

Reply:
Done in https://github.com/snek5000/snek5000/pull/209/commits/c7c1369fab7438e84547d2ebdc52cbe9a7a80431